_includes/header.html

```diff
@@ -10,37 +10,64 @@
                     <span class="icon-bar"></span>
                 </button>
                 <a class="navbar-brand" href="{{site.baseurl}}/index.html">
-                    <img src="{{site.baseurl}}/assets/nav_logo.svg" style="height:45px;">
+                    <img src="{{site.baseurl}}/assets/nav_logo.svg" style="height:44px;">
                 </a>
             </div>
             <!-- Collect the nav links, forms, and other content for toggling -->
             <div class="collapse navbar-collapse" id="bs-example-navbar-collapse-1">
                 <ul class="nav navbar-nav navbar-right">
                     {% for nav in site.data.nav.head %}
                       {% if nav.url %}
-                           <li>
-                             <a href="{{site.baseurl}}{{nav.url}}" class="tab" id="tab" target="_blank">{{nav.title | upcase}}</a>
+                           <li id="maintabs">
+                             <a href="{{site.baseurl}}{{nav.url}}" class="tab" id="tab" target="_blank">{{nav.title }}</a>
                            </li>
+                      {% elsif nav.subitem %}
+                          <li class="dropdown">
+                            <a href="#" class="dropdown-toggle" data-toggle="dropdown">{{ nav.title }} <img id="carrot" src="assets/carrot.svg" alt="carrot"></a>
```

Comment:
Same thing here. `id='carrot'` is wrong it should be a class you are in a for loop so it can appears many times.